mealie/services/scraper/recipe_scraper.py

```diff
@@ -41,9 +44,23 @@ async def scrape(self, url: str, html: str | None = None) -> tuple[Recipe, Scrap
         raw_html = html or await safe_scrape_html(url)
         for scraper_type in self.scrapers:
             scraper = scraper_type(url, self.translator, raw_html=raw_html)
-            result = await scraper.parse()
 
-            if result is not None:
-                return result
+            try:
+                result = await scraper.parse()
+            except Exception:
```

Comment:
Is it worth catching the specific exception? If not maybe throw that exception in the log so we know what the Exception is.